src/main/db/Storage.java

```diff
@@ -0,0 +1,16 @@
+package main.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> reportData = new HashMap<>();
+
+    public void setReportData(Map<String, Integer> reportData) {
+        this.reportData = reportData;
+    }
```

Comment:
setReportData replaced entirely the Storage, that is not ok. 
consider implementing addToReportData method or kind of.
```suggestion
    public void setReportData(Map<String, Integer> reportData) {
        this.reportData = reportData;
    }
```